src/components/calculationArea.tsx

```diff
@@ -102,7 +102,7 @@ const AddToCartBtn = () => {
 	return (
 		<div
 			onClick={AddToCart}
-			className="bg-red-600 text-white text-xl font-bold grid place-content-center min-h-12"
+			className="bg-red-600 text-white text-lg font-bold grid place-content-center min-h-12"
```

Comment:
At the end of this file, you have these lines:

onClick={AddToCart}

On onClick, you have passed a React Component called 'AddToCart' which is not appropriate. You should have passed a pure function even though React Component is 'technically' also a function.

Better way according to me is:
onClick={()=> alert("You successfully added the items into Cart")}

You don't need a separate 'AddToCart' component.

Reply:
1. i thought we need do a lot at the function later so that alert works as a placeholder, that's why i put like that. will change to callback if needed.
2. trying out, forgot to remove

Reply:
No.1 Please use a callback beacuse "onClick={AddToCart}" is wrong way.

No.2 is fine. Its just a different way to do the same thing. Completely okay.

Reply:
Fixed the problems accordingly, thank you for reviewing the code!